torchmetrics/text/wer/wer.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+
+from jiwer import compute_measures
```

Comment:
there need to be added a file similar to `https://github.com/PyTorchLightning/metrics/blob/master/requirements/image.txt` that specifies the required dependencies for the new text metrics

Reply:
Yes I have created a requirements/text.txt file